crates/op-rbuilder/src/integration/integration_test.rs

```diff
@@ -99,6 +104,7 @@ mod tests {
         Ok(())
     }
 
+    /// Caution: this test could be brittle to change because of queued pool checks
```

Comment:
what queued pool checks here would make the tests brittle?

Reply:
there asserts
```
                assert_eq!(
                    pool_content.queued.len(),
                    1,
                    "Queued pool should contain only 1 transaction"
                );
```
If you add/remove transactions in the tests this assert will fail